pkg/resources/management.cattle.io/v3/nodedriver/NodeDriver.md

```diff
@@ -1,7 +1,11 @@
 ## Validation Checks
 
-Note: checks only run if a node driver is being disabled or deleted
+Note: validation runs for node driver create, update, and delete operations
 
 ### Machine Deletion Prevention
 
 This admission webhook prevents the disabling or deletion of a NodeDriver if there are any Nodes that are under management by said driver. If there are _any_ nodes that use the driver the request will be denied.
+
+### Name Validation
+
+This admission webhook sanitizes the names of the NodeDriver since that name will be used in generating a Dynamic Schema.
```

Comment:
sanitizes usually infers that we are taking the name and mutating it so that it is a valid name. Instead, we are just validating that the name matches a specific format.
```suggestion
### Display Name Validation

This admission webhook validates that the names of the NodeDriver does not contain numbers or hyphens 
```